### PR TITLE
Added my italian games (they use different "Scummvm id" so they have different CRC) 

### PR DESCRIPTION
**Pull request by lollo78**
To test my commits I compiled the RDB and scanned all my games.
Works like a charm!
Thank you RobLoach for all your hard work.
:+1:

### CODE REVIEW

**Comment by RobLoach** [comment]
Rather then submitting this into the DAT directly, mind submitting it as an issue to https://github.com/robloach/libretro-scummvm.dat ? That way we can auto-generate this for all the different languages we want.

**Comment by RobLoach** [comment]
We'll have to update this to build in the `.js` file rather then updating the DAT itself. I think the way to do this would be to....
1. Introduce a `languages.json`
   
   ```
   {
       "it": "italian"
   }
   ```
2. Have it loop through each language and append `-${lang}` to each filename and content.
3. Rebuild the dat and games with the new stuffs.

**Comment by lollo78** [comment]
Ehm... 
I don't think that I understand
:D

**Comment by lollo78** [comment]
If I can help you, don't hesitate to ask ;)

File: ScummVM.dat [hunk 1]
```diff
@@ -647,6 +660,7 @@ game (
 	rom ( name "The Secret of Monkey Island.scummvm" size 6 crc b0e2af30 )
 	rom ( name "The Secret of Monkey Island Windows.scummvm" size 8 crc 8d8dbdc4 )
 	rom ( name "The Secret of Monkey Island Unix.scummvm" size 7 crc 14bed490 )
+	rom ( name "The Secret of Monkey Island (VGA - DOS - Italian).scummvm" size 14 crc f8f7bc27 )
```
Comment (RobLoach) [inline ScummVM.dat:663]: What is the short code here?

File: ScummVM.dat [hunk 2]
```diff
@@ -871,6 +886,7 @@ game (
 	rom ( name "Sam & Max Hit the Road.scummvm" size 7 crc 5b280160 )
 	rom ( name "Sam & Max Hit the Road Windows.scummvm" size 9 crc 65885b5 )
 	rom ( name "Sam & Max Hit the Road Unix.scummvm" size 8 crc 7f3e4fca )
+	rom ( name "Sam & Max Hit the Road (CD - Italian).scummvm" size 11 crc 049cb65e )
```
Comment (RobLoach) [inline ScummVM.dat:889]: What's the code for this one?

**Comment by RobLoach** [comment]
@lollo78 Just added language support in there. The ones with CD or VGA arn't in there. You may just have to edit the ID of those manually in the ScummVM interface in order to have them.

Unless you can think of a easy way to add VGA/CD support automatically to them all.

File: ScummVM.dat (hunk 1 above)
Comment (lollo78) [inline ScummVM.dat:663]: https://github.com/RobLoach/libretro-scummvm.dat/pull/5/commits/04093f67ace7e88cce2dc9fc882dc2ec152db274

File: ScummVM.dat (hunk 2 above)
Comment (lollo78) [inline ScummVM.dat:889]: https://github.com/RobLoach/libretro-scummvm.dat/pull/5/commits/9a4300f3c1c5588f7c3361d3627cba4ed16b1588

**Comment by lollo78** [comment]
support.json

```
{
    "cd": "cd"
}
```

video.json

```
{
    "vga": "vga"
}

```

?!?
lol, I don't know ;) 

@kivutar?

**Comment by RobLoach** [comment]
Adding `cd` and `vga` explicitly is an option, but then we'd end up with way too many game codes. Just informing people to fix up the ID may be the easiest. Just remove "vga" or "cd" from those in ScummVM UI.

**Comment by lollo78** [comment]
IMHO, from an user side, ScummVm is not simple like the other libretro cores.
I don't think all of them know how to remove "vga" and "cd" from ScummVM UI (or manually from the ScummVM.ini). 
Integrate this cases in your script will help a lot and will contribute to use Retroarch also for level users

**Comment by RobLoach** [comment]
@lollo78 See the [latest changes in ScummVM.dat](http://bit.ly/29KfhP7)? Makes it so that you don't need the .scummvm files anymore. You can [find the change in the ScummVM core](http://bit.ly/29PC3YK) for how it was accomplished.

**Comment by lollo78** [comment]
Hi dear! I just returned from two weeks vacation. I'll try it as soon as possible.
Thanks a lot

**Comment by inactive123** [comment]
Do you guys still want to patch this up so it can be merged?

**Comment by RobLoach** [comment]
Not entirely sure this works as expected. ScummVMs launching system is really weird, and --auto-detect will fix most of it.

Rather hold off, and seeing if the CLI launch fixes help us.

**Pull request closed without merging**